helm/ffc-pay-file-receiver/values.yaml

```diff
@@ -18,27 +18,46 @@ containerSecret:
   name: ffc-pay-file-receiver-container-secret
   type: Opaque
 
-deployment: {}
+deployment:
+  minReplicas: 1
+  maxReplicas: 1
+  cpuThreshold: 80
+  memoryThreshold: 80
+  minReadySeconds: 0
 
 container:
-  requestMemory: 100Mi
-  requestCpu: 100m
-  limitMemory: 100Mi
-  limitCpu: 100m
+  requestMemory: 120Mi
+  requestCpu: 120m
+  limitMemory: 500Mi
+  limitCpu: 500m
   port: 3000
+  messageQueueHost: namespace.servicebus.windows.net
+  fileConsumeTopicAddress: ffc-pay-file-consume
+  fileConsumeSubscriptionAddress: ffc-pay-file-consumer
+  azureStorageShareUseConnectionString: false
```

Comment:
Default should be true for this one as we can't use managed identity for DAX storage.